exercises/list-ops/canonical-data.json

```diff
@@ -71,40 +71,57 @@
         "description": "simple list",
         "list": [1,2,3,4],
         "expected": 4
-      },
+      }
     ]
   },
   "map": {
-    "description": "return a list of elements whos values equal the list value transformed by the map_function",
+    "description": "return a list of elements whos values equal the list value transformed by the mapping function",
     "cases": [
       {
         "description": "empty list",
         "list": [],
-        "map_function": "value + 1",
+        "function": "value + 1",
         "expected": []
       },
       {
         "description": "normal list",
         "list": [1,3,5,7],
-        "map_function": "value + 1",
+        "function": "value + 1",
         "expected": [2,4,6,8]
       }
     ]
   },
-  "reduce": {
-    "description": "combines list elements by applying reducer_function specified",
+  "foldl": {
+    "description": "folds (reduces) the given list from the left with a function",
+    "cases": [
+      {
+        "description": "empty list",
```

Comment:
Both folds need an "initial" parameter to supply the first input to the fold. The initial in this case would be 0.